tripal_chado/src/Plugin/Field/FieldFormatter/ChadoLocalSourceDataFormatterDefault.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Drupal\tripal_chado\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Link;
+use Drupal\Core\Url;
+use Drupal\tripal_chado\TripalField\ChadoFormatterBase;
+
+/**
+ * Plugin implementation of default Tripal string type formatter.
+ *
+ * @FieldFormatter(
+ *   id = "chado_local_source_data_formatter_default",
+ *   label = @Translation("Chado Local Source Data Formatter"),
+ *   description = @Translation("Analysis source name, version, and uri formatter"),
+ *   field_types = {
+ *     "chado_local_source_data_default"
+ *   }
+ * )
+ */
+class ChadoLocalSourceDataFormatterDefault extends ChadoFormatterBase {
+
+  /**
+  * {@inheritdoc}
+  */
+  public function viewElements(FieldItemListInterface $items, $langcode) {
+    $elements = [];
+    $elements['#attached']['library'][] = 'tripal_chado/tripal_chado.field.ChadoLocalSourceDataFormatter';
```

Comment:
I'm not sure what this library is?  Is it necessary?

Reply:
Library has been removed